Lib/base64.py

```
@@ -499,6 +497,33 @@ def b85decode(b):
         result = result[:-padding]
     return result
 
+_z85alphabet = (b'0123456789abcdefghijklmnopqrstuvwxyz'
```

Flake8 (7.2.0)

[error] expected 2 blank lines after class or function definition, found 1

(E305)